<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Priority</class>
 <widget class="QDialog" name="Priority">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>اختر الصلاحية</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string>الصلاحية</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="add_employee">
     <property name="text">
      <string>إضافه موظف</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="salary">
     <property name="text">
      <string>موظفين/راتب</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Stock">
     <property name="text">
      <string>مخزن رئيسي</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Manger">
     <property name="text">
      <string>مدير النظام</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Deliver_salary">
     <property name="text">
      <string>توزيع الرواتب</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Custmers">
     <property name="text">
      <string>العملاء</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Edit_Order">
     <property name="text">
      <string>تعديل عمليه الشراء</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Ware">
     <property name="text">
      <string>المخزن</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Open">
     <property name="text">
      <string>تشغيل</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Accounting">
     <property name="text">
      <string>حسابات</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Deliver">
     <property name="text">
      <string>تسليم</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>تم</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
